scripts/git_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Name::        git_env.sh
+# Description:: Use this script to configure local git env
+# Author::      Darren Murray (<dmurray-lacework@lacework.net>)
+#
+
+readonly commit_hook=scripts/githooks/prepare-commit-msg
+readonly hooks_path=scripts/githooks/
+readonly project_name=go-sdk
+
+prepare_git_env(){
+  chmod +x $commit_hook
+  log "Setting git-hooks path to $hooks_path"
+  git config core.hooksPath $hooks_path
```

Comment:
Changed the script to set the hooks path instead of using an update function. I also thought this might be better than symlinking each individual githook. Now to add a new hook we just need to drop it in scripts/githooks
Let me know your thoughts on this @afiune

Reply:
I'm not 100% on board to change the Git configuration of our contributors but on the other hand, this looks way cleaner now. let's just give this a try, worse case if we see problems we can always try the symlinks. 😉